share/git-process-output.zsh

```diff
@@ -11,16 +11,10 @@ if (( COLS < 10 )) {
 # Credit to molovo/revolver for the ideas
 typeset -ga progress_frames
 progress_frames=(
-  '0.2 ▹▹▹▹▹ ▸▹▹▹▹ ▹▸▹▹▹ ▹▹▸▹▹ ▹▹▹▸▹ ▹▹▹▹▸'
-  '0.2 ▁ ▃ ▄ ▅ ▆ ▇ ▆ ▅ ▄ ▃'
-  '0.2 ▏ ▎ ▍ ▌ ▋ ▊ ▉ ▊ ▋ ▌ ▍ ▎'
-  '0.2 ▖ ▘ ▝ ▗'
-  '0.2 ◢ ◣ ◤ ◥'
-  '0.2 ▌ ▀ ▐ ▄'
-  '0.2 ✶ ✸ ✹ ✺ ✹ ✷'
+  '0.1 ⠋ ⠙ ⠹ ⠸ ⠼ ⠴ ⠦ ⠧ ⠇ ⠏'
```

Comment:
That seems a bit out of scope. But I like it :)

Reply:
lol, I forgot I did this. Is out of scope, but I hated the current loading bars. 

I'll open a separate PR for this.

Reply:
This is really nice though

Reply:
Change made in PR #204